deploy/helm/llm-proxy/templates/_helpers.tpl

```diff
@@ -49,3 +49,51 @@ Selector labels
 app.kubernetes.io/name: {{ include "llm-proxy.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+{{/*
+Get the name of the secret containing MANAGEMENT_TOKEN
+*/}}
+{{- define "llm-proxy.managementTokenSecretName" -}}
+{{- if and .Values.secrets.create .Values.secrets.data.managementToken }}
+{{- include "llm-proxy.fullname" . }}
+{{- else if .Values.secrets.managementToken.existingSecret.name }}
+{{- .Values.secrets.managementToken.existingSecret.name }}
+{{- else }}
+{{- printf "" }}
```

Comment:
Nit: `printf ""` is redundant here; returning an empty string can just be `""` (or omit the else branch). More importantly, this is a good place to centralize validation (e.g., `required` for MANAGEMENT_TOKEN secret name when running in production mode).